src/main/java/com/tstepnik/planner/domain/task/TaskDto.java

```diff
@@ -0,0 +1,40 @@
+package com.tstepnik.planner.domain.task;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.validation.constraints.NotEmpty;
+
+public class TaskDto {
```

Comment:
I personally like to name dto's like this: `TaskDTO` instead of `TaskDto` because dto part is just more prominent, but it is up to you which one you want to use :)

Reply:
I also wanted do that but I wasn't sure it's correct with good pracices. I will change it.